.github/workflows/security-center-snippets-v2.yaml

```diff
@@ -0,0 +1,101 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: security-center-snippets-v2
+on:
+  push:
+    branches:
+    - main
+    paths:
+    - 'security-center/snippets/v1/**'
```

Comment:
suggestion: Should this be targeting v2?

Reply:
I caught that and force-pushed already; beat you to it :)